bite-size-notes/week-5-c-Reachability-Analysis.md

```diff
@@ -0,0 +1,215 @@
+# Reachability Analysis
+
+## Scope+Objectives
+This bite-sized note will cover Reachability Analysis from our week 5 lecture. It will focus on three key concepts: forward reachability, backward reachability, and the Hamilton-Jacobi reachability methods.
+
+## Introduction
```

Comment:
you can add more context about why reachability analysis is crucial in control systems. Perhaps include examples of applications to give readers a better grasp of their importance.